portfolio/src/main/webapp/style.css

```diff
@@ -13,3 +13,12 @@
 .comment span {
   flex-grow: 1;
 }
+
+#map {
+  height: 500px;
+  width: 500px;
+  margin: 0 auto;
+  border: 10px solid black;
+  background-color: grey;
+}
+ 
```

Comment:
Showing an empty line, but also the no new line marker. Maybe last line has white space on it?